frontend/src/services/FirebaseService.ts

```diff
@@ -1,22 +1,14 @@
 import * as firebase from "firebase/app";
 import "firebase/auth";
 
-// const FIREBASE_KEY = process.env.FIREBASE_KEY;
-// const FIREBASE_AUTH_DOMAIN = process.env.FIREBASE_AUTH_DOMAIN;
-// const FIREBASE_DATABASE_URL = process.env.FIREBASE_DATABASE_URL;
-// const FIREBASE_PROJECT_ID = process.env.FIREBASE_PROJECT_ID;
-// const FIREBASE_STORAGE_BUCKET = process.env.FIREBASE_STORAGE_BUCKET;
-// const FIREBASE_MESSAGING_SENDER_ID = process.env.FIREBASE_MESSAGING_SENDER_ID;
-// const FIREBASE_APP_ID = process.env.FIREBASE_APP_ID;
-// const FIREBASE_MEASUREMENT_ID = process.env.FIREBASE_MEASUREMENT_ID;
-
-const FIREBASE_KEY = "AIzaSyCkhlUKdbkaMnorgs4OeQ0-pro93v-rWHo";
```

Comment:
Let's be sure to rotate this key in Firebase

Reply:
Yes I set up a new project

Reply:
Great, thanks for looking into that.